UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamage.cs

```diff
@@ -181,9 +187,9 @@ public partial class BodyPart
 		public float RadiationStacks => Damages[(int) DamageType.Radiation];
 
 
-		[HideInInspector] public float CurrentInternalBleedingDamage
-		{
-			get
+		[HideInInspector] public float CurrentInternalBleedingDamage 
```

Comment:
properties don't show in inspector, you don't need to hide them

Reply:
fixed